matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/crypto/store/db/model/IncomingRoomKeyRequestEntity.kt

```diff
@@ -33,9 +33,9 @@ internal open class IncomingRoomKeyRequestEntity(
 
     fun toIncomingRoomKeyRequest(): IncomingRoomKeyRequest {
         return IncomingRoomKeyRequest().apply {
-            requestId = requestId
-            userId = userId
-            deviceId = deviceId
+            requestId = this@IncomingRoomKeyRequestEntity.requestId
+            userId = this@IncomingRoomKeyRequestEntity.userId
+            deviceId = this@IncomingRoomKeyRequestEntity.deviceId
```

Comment:
Keep my version please, sorry for the conflict

Reply:
ok i reverted to error code, will be fixed with other merge